src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -409,3 +389,124 @@ def get_policy_assignment_completion_list(prefix, **kwargs):#pylint: disable=unu
     result = policy_client.policy_assignments.list()
     return [i.name for i in result]
 
+class _ResourceUtils(object): #pylint: disable=too-many-instance-attributes
+    def __init__(self, resource_group_name=None, resource_provider_namespace=None,
```

Comment:
By default we don't want to validate input for API which is not LRO and server error is clear. But for this command, we do validation when api-version is missing, which we expect would be the majority case